### PR TITLE
Review

### PR DESCRIPTION
**Pull request by Javiercerna**

### CODE REVIEW

**Review by Javiercerna — COMMENTED** [review]
In general, the code looks good! But I can't help but notice that the functionality is not finished, since for the second node we want both a subscriber and a publisher. Please be careful while reading the instructions :slightly_smiling_face: 

You don't need to implement anything now, but think how you would do it and you can explain it during the interview.

File: publisher.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+import rospy
+import std_msgs
+
+
+def publisher():
+
+    # INITIALIZATION
+    pub = rospy.Publisher('/albert', std_msgs.msg.UInt32, queue_size=10) # topic being published to and type
```
[the remark this answers is not in the document]
Comment (BridgeQZH) [reply publisher.py:9]: Agree!

File: subscriber.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+import rospy
+import std_msgs
+
+def callback(data):
+    q = 0.15
+    print(data.data/q) # data.data accesses the value of the integer in the message
```
[the remark this answers is not in the document]
Comment (BridgeQZH) [reply subscriber.py:8]: You need to publish the result to a new topic. And you need to be careful with the datatype used by that rostopic.

[the remark this answers is not in the document]
Comment (BridgeQZH) [reply subscriber.py:8]: And consider whether to put the publisher in the callback function.

File: publisher.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    n = 4
+
+    # COMPUTATION 
+    while not rospy.is_shutdown():
+        k += n
+        message = std_msgs.msg.UInt32(k) # Create the message
+        pub.publish(message) # Publish the message
+        rate.sleep() # Wait a bit to send the next message
+        
+
```
Comment (BridgeQZH) [inline publisher.py:23]: Delete a line

**Pull request closed without merging**